supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -39,6 +39,25 @@ Some tasks in this procedure require root privileges, which you can get temporar
 .Additional resources
 * link:https://www.redhat.com/sysadmin/difference-between-sudo-su[Exploring the differences between sudo and su commands in Linux]
 
+[[ellipses-in-yaml-code-blocks]]
```

Comment:
Should this section be moved to the **Formatting** section instead of the **Technical examples** section?

Reply:
I wasn't sure that this was formatting. The topic seemed closer to 'syntax highlighting'.